CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 7.0.0 - 2022-07-19
+Decommission Trip Parser API V2 
```

Comment:
Hey @minjikarin, could you change the capitals of V2 and V3 to small for this one and last release updates? In the changelogs. In changelogs and blog articles we write it with small v, so let's keep it the same for consistency.